engine/api/services/http.go

```diff
@@ -89,7 +89,7 @@ func doStreamRequest(ctx context.Context, srvs []sdk.Service, method, path strin
 			srv := &srvs[i]
 			reader, headers, code, err := doStreamFromURL(ctx, srv, method, path, bytes.NewReader(b), mods...)
 			if err != nil {
-				lastErr = err
+				lastErr = sdk.WrapError(err, "service:%v", srv.Name)
```

Comment:
you should add metadata as string in the error message, but on the context. The lastErr is print in this function

Reply:
log not really useful in fact. (and can't use field in the context, it'a a range of services, and we print err after the loop)